CHANGELOG.md

```diff
@@ -142,6 +142,23 @@ Performance
   Meta-variables are saved if the pragma option `--save-metas` is
   used. This option can be overridden by `--no-save-metas`.
 
+* The new option `--syntactic-equality[=FUEL]` can be used to limit
+  how many times the syntactic equality shortcut is allowed to fail
```

Comment:
> how many times

This should detail the "per": per file, per invokation of the conversion rule, per ...

Reply:
See below: "Currently the fuel is decreased in the failure continuations of the implementation of the syntactic equality shortcut." I did not include this explanation in the user manual, because it is tied to the implementation, and I do not know the details of the implementation well enough to give a more high-level explanation.

Reply:
Now I understand what you mean. Would the following make more sense to you? "Currently the fuel is decreased in the failure continuations of the implementation of the syntactic equality shortcut. When a failure continuation completes the fuel is restored to its previous amount."

Reply:
Ok, this helps.
Also, I suppose each invocation of the syntactic equality check from the outside, i.e., the conversion check, starts with a full tank?

Reply:
Is the conversion checker ever invoked from the failure continuation of the syntactic equality check?

I added some text to the explanation.